### DISCUSSION TITLE
How to make range breaks of financial data?

### DISCUSSION THREAD

**Discussion by yangguang760** [GSI-CS-CO/chart-fx#336] [Q&A] [answered]
For example , 9:00 am - 11:00 am and 13:00 pm -15:00pm.

**Comment by raven2cz** [comment, marked as answer]
Hi,
in actual master branch is implemented usage of XAxis based on time only.
It is this line:  `xAxis.setTimeAxis(true)`
With time based axis is not possible to easily use some time range breaks.
The fin. renderers support CategoryAxis where is possible to place ohlc by its indexes, the xaxis is depended on the indexes which is better for your problem.
But CategoryAxis is not good approach for visualization of financial time axis data. We are planning some "marriage" between time and indexes, like TimeIndicesAxis. It means that the axis takes indices of the ohlc and indicators but have great possibilities for aggregation and visualization of time labels which are mapped to this data too. This is not implemented yet.

**Reply by yangguang760** [reply]
Thanks for your kindly reply. 
Maybe the implementation of Plotly, a famous Javascript plot program, could offer some “marriage” ways.
